docs/usage.md

```diff
@@ -71,46 +71,72 @@ In addition to the cloud deployment, we provide an opportunity to launch benchma
 This allows us to conduct debugging and a local characterization of the benchmarks.
 
 First, launch a storage instance. The command below is going to deploy a Docker container,
-map the container's port to port `9011` on host network, and write storage instance configuration
-to file `out_storage.json`
+map the container's port to port defined in the configuration on host network, and write storage 
+instance configuration to file `out_storage.json`
 
-```
-./sebs.py storage start minio --port 9011 --output-json out_storage.json
+```bash
+./sebs.py storage start all config/storage.json --output-json out_storage.json
 ```
 
 Then, we need to update the configuration of `local` deployment with information on the storage 
 instance. The `.deployment.local` object in the configuration JSON must contain a new object
 `storage`, with the data provided in the `out_storage.json` file. Fortunately, we can achieve
 this automatically with a single command by using `jq`:
 
-```
-jq '.deployment.local.storage = input' config/example.json out_storage.json > config/local_deployment.json
+```bash
+jq '.deployment.local.storage = input' config/example.json out_storage.json | \
+jq '.experiments.architecture = "x64"' > config/local_deployment.json
```

Comment:
Better idea - please add architecture to `local` part of  `config/example.json`?

Reply:
Currently the suite doesn't read architecture from deployment section (the `local` part) of the configuration. Would you like me to add a logic that architecture defined in deployment section overwriting architecture defined in experiments? Or another workaround is sepcifying architecture when starting the function containers: `./sebs.py local start ... --architecture=x64`.

Reply:
We can use the CLI argument :)